Utilities/ci/azure-pipelines.yml

```diff
@@ -47,8 +50,8 @@ jobs:
       testRunner: JUnit
       testResultsFiles: 'Utilities/TestResults/TESTS-*.xml'
 
-- job: 'Build'
-  displayName: 'Build'
+- job: 'BuildTestMacOS'
+  displayName: 'Build and test'
```

Comment:
-> *Build and test* -> *Build* (we are only running the tests on Linux because that is where we have a WebGL environment).